spark/src/main/scala/geotrellis/spark/RasterMetaData.scala

```diff
@@ -2,26 +2,37 @@ package geotrellis.spark
 
 import geotrellis.raster._
 import geotrellis.spark.tiling._
-import geotrellis.vector.Extent
+import geotrellis.vector.{ProjectedExtent, Extent}
 
 import geotrellis.proj4.CRS
 
 import org.apache.spark.rdd._
 
+/**
+ * @param cellType    value type of each cell
+ * @param layout      definition of the tiled raster layout
+ * @param extent      extent covering the source data cells
+ * @param crs         CRS of the raster projection
+ */
 case class RasterMetaData(
   cellType: CellType,
+  layout: LayoutDefinition,
   extent: Extent,
-  crs: CRS,
-  tileLayout: TileLayout
+  crs: CRS
 ) {
+  /** Transformations between tiling scheme and map references */
+  def mapTransform = layout.mapTransform
+  /** Layout raster extent */
+  def rasterExtent = layout.rasterExtent
+  /** TileLayout of the layout */
+  def tileLayout = layout.tileLayout
+  /** Full extent of the layout */
+  def layoutExtent = layout.extent
```

Comment:
Is there a huge need to have these delegation methods? I'm usually opposed to having the shortcut method, because it makes refactoring twice as hard, and there's more explicit ownership when only one type has the method.